crates/orchestrator/src/main.rs

```diff
@@ -14,14 +10,7 @@ use utils::env_utils::get_env_var_or_default;
 // version but have added it for now
 #[allow(clippy::needless_return)]
 async fn main() {
-    dotenv().ok();
```

Comment:
let's keep `dotenv().ok();` on start of main instead of `setup_analytics`

Reply:
Alright, [shifted](https://github.com/madara-alliance/madara-orchestrator/pull/143/commits/a8d3e114b75add369d8ae8c7061ed9b741b4b4ba).